bindings/nodejs/tests/closestSoletta.js

```diff
@@ -0,0 +1,19 @@
+var fs = require( "fs" );
+var path = require( "path" );
+var async = require( "async" );
+var dirToCheck;
+
+// Find the closest instance of soletta via its package.json
+function closestSoletta( startDir ) {
+	for ( dirToCheck = startDir; dirToCheck !== "/";
+			dirToCheck = path.resolve( dirToCheck, ".." ) ) {
+		try {
+			if ( JSON.parse( fs.readFileSync(
+					path.join( dirToCheck, "package.json" ), "utf8" ) ).name == "soletta" ) {
+				return dirToCheck;
+			}
+		} catch( errors ) {}
```

Comment:
An error is actually expected here and this is the correct way to handle it. An error indicates that a ```package.json``` file was not found at that location, so you must keep searching up the tree. You don't want, like
```
Failed to open /home/nix/iot/soletta/tests/tests/package.json
Failed to open /home/nix/iot/soletta/tests/package.json
```
and then the test works fine, because it finds ```package.json``` at the next location up the tree, ```/home/nix/iot/soletta```.

```console.log()``` cannot be used in tests, because tests run as child processes of ```suite.js``` which expects their stdout to be a sequence of well-formed JSON objects. Thus, ```console.error()``` is our only recourse for arbitrary output - but again, I wouldn't output anything in this case, because a failure is very much expected.